code/modules/clothing/suits/armor.dm

```diff
@@ -205,6 +205,19 @@
 
 			return PROJECTILE_CONTINUE // complete projectile permutation
 
+/obj/item/clothing/suit/armor/greatcoat/german_overcoat
+	name = "black german overcoat"
+	desc = "A black german overcoat with armored plates"
```

Comment:
You need to mention that it made from special materials that negate energy damage better.